src/components/Empty/SVG/DefaultEmptyDataImg.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+export const DefaultEmptyDataImg = (): JSX.Element => {
+    return (
+        <svg
+            className={'empty-data-image-default'}
+            width="200"
+            height="184"
+            viewBox="0 0 200 184"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <path
+                d="M39.0718 177.839C21.8078 171.861 3.66074 162.325 0.510726 150.175C-2.63929 138.025 9.18787 123.359 24.8068 115.637C40.5358 108.029 59.9069 107.449 71.9662 102.257C83.9156 96.9498 88.5731 86.9309 99.9189 77.9985C111.265 69.066 129.299 61.2199 140.372 64.9517C151.555 68.798 155.648 84.2069 164.716 96.6426C173.785 109.078 187.68 118.625 194.904 131.657C202.237 144.804 202.768 161.422 189.061 166.219C175.355 171.016 147.3 163.877 128.36 165.625C109.569 167.288 99.7624 177.823 86.4541 181.955C73.2954 186.004 56.4854 183.734 39.0718 177.839Z"
+                fill="#EBF7FF"
+            />
+            <path d="M26 108L38.5 59H160.5L174 108V154H26V108Z" fill="white" />
+            <path
+                fillRule="evenodd"
+                clipRule="evenodd"
+                d="M118.495 108.258C118.495 117.504 110.214 123.333 99.9993 123.333C89.7843 123.333 81.5034 117.504 81.5034 108.258C81.5034 107.958 81.5121 105.994 81.5293 105.698H46.666L60.2683 72.6258C60.8547 71.0563 62.4813 70 64.3116 70H135.687C137.517 70 139.144 71.0563 139.73 72.6258L153.333 105.698H118.469C118.487 105.994 118.495 107.958 118.495 108.258Z"
+                fill="#B0F3FE"
```

Comment:
we are going to need a mechanism by which we apply svg fills via class names so we can support theming of svgs.  It might be the case that we want these empty state graphics to change colors when switching to dark mode.

Reply:
This can be done via props passed to the SVG component. Can also be done in SASS like the other components via the theme selectors